### ISSUE TITLE
Chrome extension

### ISSUE THREAD

**Issue by OmgImAlexis** [eduardoboucas/compat-report#9]
@eduardoboucas can you please close this once the Chrome version is out.

Adding this issue so people can subscribe.

**Comment by eduardoboucas** [comment]
Definitely. The current build _should_ be compatible with Chrome (you can try the development version). I just need to find out what exactly is involved in publishing the extension to the Chrome Web Store.

Thanks!

**Comment by entepe85** [comment]
The current build seems to be perfectly compatible with Chrome. I just downloaded this repo as a zip, unpacked it and loaded the dist/ folder into Chrome using the extension development mode which allows the loading of unpacked extensions.

**Comment by a7madgamal** [comment]
I would love to get this in chrome too :)
